redis_commands.c

```diff
@@ -3052,14 +3061,6 @@ int redis_blpop_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
         "BLPOP", sizeof("BLPOP")-1, 2, 1, cmd, cmd_len, slot);
 }
 
-/* BRPOP */
-int redis_brpop_cmd(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
-                    char **cmd, int *cmd_len, short *slot, void **ctx)
-{
-    return gen_varkey_cmd(INTERNAL_FUNCTION_PARAM_PASSTHRU, redis_sock,
-        "BRPOP", sizeof("BRPOP")-1, 1, 1, cmd, cmd_len, slot);
```

Comment:
`min_argc` here is 1 but in `redis_varkey_timeout_cmd` you use 2. Where is th bug? :smile:

Reply:
Good catch I'll take a look.

Reply:
I think the bug was in the old code.  All blocking pops seem to require at least one key and a timeout as far as I can tell.

https://redis.io/commands/BLPOP
https://redis.io/commands/bzpopmin